frontend/packages/dev-console/src/components/pipelines/modals/common/SecretsList.scss

```diff
@@ -1,5 +1,5 @@
 .odc-secrets-list {
   &__secretsItem {
```

Comment:
```suggestion
```

You dropped the block level, need to move this to up from the element level to replace it.